tests/testthat/error/test-rep.txt

```diff
@@ -3,8 +3,7 @@
 =============================
 
 > vec_rep(1, "x")
-Error: Can't convert from `times` <character> to <integer> due to loss of precision.
-* Locations: 1
+Error: Can't convert from `times` <character> to <integer>.
```

Comment:
Not related to this PR but shouldn't it be "Can't convert `times` from <character> to <integer>."?

Reply:
Noted in https://github.com/r-lib/vctrs/issues/922#issuecomment-605417304